doc-content/enterprise-only/cases/case-management-creating-order-placed-milestone-proc.adoc

```diff
@@ -3,32 +3,31 @@
 This milestone is reached when the `ordered` variable, which is part of the *Place order* sub-process, has been given a response.
 
 .Procedure
-. In the process designer, click the default *Milestone* object.
-. In the upper-right corner, click image:cases/left-arrows.png[] to open the *Properties* panel.
+. In the process designer, expand *Milestone* in the *Object Library* and drag a new milestone on the canvas below the *Notify requestor* script task.
+. Click the new milestone and in the upper-right corner click the *Properties* image:getting-started/diagram_properties.png[] icon.
 . Input `Milestone 1: Order placed` in the *Name* field.
-. Select `true` from the *Adhoc autostart* menu.
-. Click the down arrow in the *Assignments* field to add a new condition in the *Milestone 1: Order placed Data I/O* window.
-. Click the *Source* column drop-down, select *Constant*, and input `org.kie.api.runtime.process.CaseData(data.get("ordered") == true)`. This means that a case variable named `ordered` exists with the value `true`.
+. Expand *Implementation/Execution* and select *Adhoc Autostart*.
+. Expand *Data Assignments*, click image:getting-started/diagram_properties.png[] in the *Assignments* field, and add the following:
 +
 image::cases/milestone-evaluation-condition.png[Milestone evaluation condition configuration]
 +
-
+Click the *Source* column drop-down, select *Constant*, and input `org.kie.api.runtime.process.CaseData(data.get("ordered") == true)`. This means that a case variable named `ordered` exists with the value `true`.
 . Click *Save*.
 . Click *Milestone 1: Order placed* and create a new script task.
 +
 image::cases/new-task.png[new task creation]
 +
 image::cases/script-task.png[convert to a script task]
 
-. Click the new script task and in the upper-right corner, click image:cases/left-arrows.png[] to open the *Properties* panel.
+. Click the new script task and in the upper-right corner click the *Properties* image:getting-started/diagram_properties.png[] icon.
 . Input `Notify requestor` in the *Name* field.
-. Click the down arrow in the *Script* field, input `System.out.println("Notification::Order placed");`, and click *Ok*.
+. Expand *Implementation/Execution* and input `System.out.println("Notification::Order placed");`.
 . Click the *Notify requestor* script task and create a signal end event.
-+
-image::cases/end-event.png[new end event]
-+
-image::cases/signal-event.png[convert to a signal event]
-
-. Click the signal event and click image:cases/left-arrows.png[] to open the *Properties* panel.
+. Click the signal event and in the upper-right corner click the *Properties*.
+ image:getting-started/diagram_properties.png[] icon.
+. Expand *Implementation/Execution*, click the down arrow in the *Signal* field, and select `Milestone 2: Order shipped`.
```

Comment:
I believe this won't work, as "Milestone 2: Order shipped" wouldn't be in the dropdown (only when it's already been defined elsewhere, but I don't see where that would have been)?  So you would have to select "New value ..." and then type the name. But there's actually a bigger issue here, as there's a known issue https://issues.redhat.com/browse/RHPAM-2557 that prevents you from using spaces or special characters in signal names.  That's why I suggested before to simply use "Milestone2" and "Milestone3" as names of the milestone nodes (and therefore also as signal names) to work around this issue.

Reply:
Hi @krisv. Yes, I am aware of this workaround (aka - Known issue). Due to the rush to complete this doc, I utilized the sample for screen captures. I think we can add a note at the beginning of the process about the know issue and advise readers to modify the milestone names to fit the criteria until the issue is resolved. Do you agree? Frankly, post holiday break, it is a struggle to walk through an entire GSG process again. I have done a few already. If you disagree, then I will do what has to be done to make it happen, but it will take time.